### ISSUE TITLE
CP hidden on closed non-binary questions

### ISSUE THREAD

**Issue by jkraybill** [Metaculus/metaculus#1756]
## Current behavior
Two questions from the AIB tournament closed yesterday, but the community prediction is not being shown.

[30927](https://www.metaculus.com/questions/30927/):

<img width="570" alt="image" src="https://github.com/user-attachments/assets/8edf9f75-fdda-4b4d-ba2c-5c698ee0e4cf" />

[31010](https://www.metaculus.com/questions/31010/most-tweets-dec-21-30-24-elon-musk-nyt-or-the-economist/):

<img width="571" alt="image" src="https://github.com/user-attachments/assets/24822bb3-08c3-4f15-9d7d-522778db95ed" />

These are the first non-binary questions in the AIB tournament, so I suspect question type is a factor.

## Expected/desired behavior
I expect to see the chart and details of number of predictors and community prediction, like I do for binary questions.

**Comment by Tommetaculus** [comment]
I concur that this is a bug.

**Comment by lsabor** [comment]
> a data problem with how the questions were launched

Yes, that's almost certainly what's going on. If @elisescu can't get to it today, I probably can.

>will this be resolved for future non-binary questions on the AIB

Requires tweaks to the launcher (hence calling in Elis)

> can these already launched questions have this resolved

Yes. I think all that needs to happen is to go on the admin panel, check the `include_bots_in_aggregates` field, and save. I believe this triggers a recalc of the CP. If it doesn't, let me know and I'll go in and manually trigger the recalculations.
If any of these questions have resolved, they will have to be unresolved and then re-resolved.

**Comment by Tommetaculus** [comment]
@lsabor I edited it in admin, so bots are included - but the CP didn't update. I didn't see a way to trigger that.

https://www.metaculus.com/questions/30927/what-will-be-the-domestic-earnings-of-a-complete-unknown/

![Screenshot 2024-12-19 at 3 51 28 PM](https://github.com/user-attachments/assets/ae3192d7-8fab-48d9-b617-b51e1eb4f85e)

**Comment by Tommetaculus** [comment]
Elis's upgrades to the admin tool work - they include bots in aggregate - but these questions were launched manually, since the tool wasn't ready at the time.
